tests/plugins/test_plugins_manager.py

```diff
@@ -94,59 +95,53 @@ def test_entrypoint_plugin_errors_dont_raise_exceptions(self, mock_ep_plugins):
 
 
 class TestPluginsManager(unittest.TestCase):
-    class AirflowTestPropertyPlugin(AirflowPlugin):
```

Comment:
I moved this class to the function to maintain one style of code. It's easier to find the right test when all the code is in one place.